ServerCore/Pages/Events/Import.cshtml.cs

```diff
@@ -0,0 +1,155 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using Microsoft.EntityFrameworkCore;
+using ServerCore.DataModel;
+
+namespace ServerCore.Pages.Events
+{
+    public class ImportModel : PageModel
+    {
+        private readonly PuzzleServerContext _context;
+
+        public ImportModel(PuzzleServerContext context)
+        {
+            _context = context;
+        }
+
+        [BindProperty]
+        public Event Event { get; set; }
+
+        public IList<Event> Events { get; set; }
+
+        [BindProperty]
+        public int ImportEventID { get; set; }
+
+        public async Task<IActionResult> OnGetAsync(int id)
+        {
+            Event = await _context.Events.SingleOrDefaultAsync(m => m.ID == id);
+
+            if (Event == null)
+            {
+                return NotFound();
+            }
+
+            Events = await _context.Events.Where(e => e != Event).ToListAsync();
```

Comment:
It might be nice to put in an IsAdminForEvent check for the other events to avoid picking illegal ones